install/deploy_final_non-compute_node.md

```diff
@@ -154,43 +154,25 @@ The steps in this section load hand-off data before a later procedure reboots th
             /var/log/conman \
             /var/log/zypper.log 2>/dev/null |
          sed 's_^/__' |
-         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$(date +%Y-%m-%d_%H-%M-%S).tgz"
+         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$PITBackupDateTime.tgz"
     ```
 
-1. (`pit#`) Backup the bootstrap information from `ncn-m001`.
+1. (`pit#`) Backup the bootstrap information from `ncn-m001` before reboot.
 
-    > **`NOTE`** This preserves information that should always be kept together in order to fresh-install the system again.
+    > **`NOTE`** This preserves information that should always be kept together. This data can be critical for debug and support as well as in order to fresh-install the system again.
 
-    1. Log in and set up passwordless SSH **to** the PIT node.
+   ```bash
+   PITBackupDateTime=$(date +%Y-%m-%d_%H-%M-%S); cray artifacts create config-data pit-logs-backup-$PITBackupDateTime.tgz ${PITDATA}/prep/logs/pit-backup-$PITBackupDateTime.tgz
+   ```
 
-        Copying **only** the public keys from `ncn-m002` and `ncn-m003` to the PIT node. **Do not** set up
-        passwordless SSH **from** the PIT node or the key will have to be securely tracked or expunged if using a USB installation).
+1. Tar and backup the full contents of prep and configs tarball to the config-data bucket as well.
```

Comment:
If this command is uploading the entire contents of `prep`, then why did we upload that one file from a subdirectory of `prep` in the previous step?

Reply:
Thats a valid point and thought of it as well. The previously written docs told the user to upload just the logs, where as I think it will be better if we had all of prep, configs and logs directories. I just didn't simply and integrate the previous steps. In actuality I just want to put this whole thing into a simple script and tell someone to run the script, but that will likely come later.

Reply:
Removed the direct upload of the logs tar ball,  and just left the upload for the whole configs and prep directory